### PR TITLE
Use melos to handle monorepo

### PR DESCRIPTION
**Pull request by nielsenko**
Add support for [melos](https://melos.invertase.dev/) to make it easy to setup dependency overrides.
```sh
dart pub global activate melos # one-time setup to activate melos
```
Now add `~/.pub-cache/bin` if you haven't already.

To setup dependency overrides run `melos bootstrap`,

Also added a few common scrips:
```sh
melos analyze
melos format:check
melos test
```
and `melos format`, but be aware that the last will format all your code with `dart format`. It seems you are currently working with another linelength than 80.

### CODE REVIEW

File: packages/passkeys/passkeys/example/lib/main.dart [hunk 1]
```diff
@@ -12,21 +12,6 @@ class MyApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
       home: HomePage(),
-      theme: ThemeData(
```
Comment (incorbador) [inline packages/passkeys/passkeys/example/lib/main.dart:15]: I guess that wasn't commited on purpose, right? :)

**Review by incorbador — CHANGES_REQUESTED** [review]
Cool tool, is going to make a lot of sense for us. 
Especially as soon as we are adding tests and enforcing some formating (this is on our short term list). 

I'm happy to merge it now, just remove the theming change please.

File: packages/passkeys/passkeys/example/lib/main.dart (hunk 1 above)
Comment (nielsenko) [reply packages/passkeys/passkeys/example/lib/main.dart]: Yes, that wasn't on purpose. I'll revert it.

Comment (nielsenko) [reply packages/passkeys/passkeys/example/lib/main.dart]: Done

**Review by incorbador — APPROVED** [review]

**Pull request merged**